crypto/curve25519/spake25519.c

```diff
@@ -384,7 +384,8 @@ int SPAKE2_generate_msg(SPAKE2_CTX *ctx, uint8_t *out, size_t *out_len,
   // bit and so one for all the bottom three bits.
 
   scalar password_scalar;
-  OPENSSL_memcpy(&password_scalar, password_tmp, sizeof(password_scalar));
```

Comment:
there's also a memcpy to `ctx->password_hash` on L369. is that effected by endianness?